### ISSUE TITLE
Why Does Ə Look Like O?

### ISSUE THREAD

**Issue by Bronnel721** [cyrealtype/Inconsolata#81]
![Screenshot (46)](https://user-images.githubusercontent.com/92538982/192076205-ff12a53f-d8c7-4eac-a992-7df2e5d9d28f.png)

**Comment by CrendKing** [comment]
Don't know why, but if you are on the writing end, you could try `ə` instead?

**Comment by Bronnel721** [comment]
@CrendKing Yes, I'm talking about in words that feature that word as the first letter that is either alone or at the start of a sentence.
